src/articles/quick-start/04-playbooks.md

```diff
@@ -0,0 +1,86 @@
+---
+uid: quick-start.playbooks
+---
+
+# Create your first Playbook
+
+Now that you've [created a customer record](xref:quick-start.customer-setup), let's create a Playbook to notify our team when any of our customers appear to be unhappy.
+
+## Create a Playbook
+
+Playbooks are a way to automate common Customer Operations workflows.
+See the [Playbooks](xref:playbooks) page for more information.
+For now, we'll create a simple Playbook that notifies our team when a customer appears to be unhappy.
+
+1. Start on the [Playbooks page in Abbot](https://app.ab.bot/Playbooks). Click "Create" to create a new Playbook.
+
+    <img src="/public/images/articles/quick-start.Playbooks/create-Playbook-button.png">
```

Comment:
Lots of broken images on this page.
```suggestion
    <img src="/public/images/articles/quick-start.playbooks/create-playbook-button.png">
```

Reply:
I did a find-replace goof 😭

Reply:
Unfortunately, the broken image detector script doesn't detect this issue because APFS is case-insensitive (but case-preserving). So the script "works". I'm hesitant to invest a bunch more time in fixing it, hopefully case issues will be rare.